website/content/en/docs/deployment/cognito/manifest/guide-automated.md

```diff
@@ -6,137 +6,68 @@ weight = 10
-                    certARN: arn:aws:acm:us-east-1:123456789012:certificate/9d8c4bbc-3b02-4a48-8c7d-d91441c6e5af
-                    hostedZoneId: XXXXX
-                    name: example.com
-                subDomain:
-                    us-west-2-certARN: arn:aws:acm:us-west-2:123456789012:certificate/d1d7b641c238-4bc7-f525-b7bf-373cc726
-                    hostedZoneId: XXXXX
-                    name: platform.example.com
-                    us-east-1-certARN: arn:aws:acm:us-east-1:123456789012:certificate/373cc726-f525-4bc7-b7bf-d1d7b641c238
-            ```
-
-## 3.0 Install Kubeflow
-1. Install Kubeflow using the following command:
-{{< tabpane persistLang=false >}}
-{{< tab header="Kustomize" lang="toml" >}}
-make deploy-kubeflow INSTALLATION_OPTION=kustomize DEPLOYMENT_OPTION=cognito
-{{< /tab >}}
-{{< tab header="Helm" lang="yaml" >}}
-make deploy-kubeflow INSTALLATION_OPTION=helm DEPLOYMENT_OPTION=cognito
-{{< /tab >}}
-{{< /tabpane >}}
+The following steps automate:
 
-## 4.0 Updating the domain with ALB address
-1. Check if ALB is provisioned. It takes around 3-5 minutes
-    1. ```sh
-        kubectl get ingress -n istio-system
-        Warning: extensions/v1beta1 Ingress is deprecated in v1.14+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
-        NAME            CLASS    HOSTS   ADDRESS                                                              PORTS   AGE
-        istio-ingress   <none>   *       k8s-istiosys-istioing-xxxxxx-110050202.us-west-2.elb.amazonaws.com   80      15d
-        ```
-    2. If `ADDRESS` is empty after a few minutes, check the logs of alb-ingress-controller by following [this guide]({{< ref "/docs/troubleshooting-aws.md#alb-fails-to-provision" >}})
-1. Substitute the ALB address under `kubeflow.alb.dns` in `tests/e2e/utils/cognito_bootstrap/config.yaml`. The kubeflow section of the config file will look like:
-    1. ```yaml
-        kubeflow:
-            alb:
-                dns: ebde55ee-istiosystem-istio-2af2-1100502020.us-west-2.elb.amazonaws.com
-                serviceAccount:
-                    name: alb-ingress-controller
-                    policyArn: arn:aws:iam::123456789012:policy/alb_ingress_controller_kube-eks-clusterxxx
-        ```
-1. Run the following script to update the subdomain with ALB address
-    1. ```sh
-        cd tests/e2e
-        PYTHONPATH=.. python utils/cognito_bootstrap/cognito_post_deployment.py
-        cd -
-        ```
-## 5.0 Connect to central dashboard 
-[section 6.0(Connecting to central dashboard)]({{< ref "/docs/deployment/cognito/manifest/guide.md#60-connecting-to-central-dashboard" >}}) to:
-    1. Create a user in Cognito user pool
-    1. Create a profile for the user from the user pool
-    1. Connect to the central dashboard
+- the creation of a [custom domain and certificates]({{< ref "/docs/deployment/cognito/manifest/guide.md#10-custom-domain-and-certificates" >}}) to host Kubeflow and TLS certificates for the domain.
```

Comment:
first 2 links are broken for the bullet points, can we say:
- the creation of .<blah>. as in section 1.0 of manual deployment guide to ...